diff --git a/haskell-src.cabal b/haskell-src.cabal
index ad99aef..eace1d4 100644
--- a/haskell-src.cabal
+++ b/haskell-src.cabal
@@ -36,7 +36,7 @@ library
         Language.Haskell.Syntax,
         Language.Haskell.ParseUtils
 
-  build-depends: base    >= 4.3 && < 4.14
+  build-depends: base    >= 4.3 && < 4.15
                , syb     >= 0.1 && < 0.8
                , pretty  >= 1.0.1.2 && < 1.2
                , array   >= 0.3 && < 0.6
